swan-lake/resources/featured-scenarios/build-a-data-service-in-ballerina.md

```diff
@@ -59,222 +55,257 @@ CREATE TABLE Company.Employees (
+    ```toml
+    USER="root"
+    PASSWORD="rootPassword"
+    HOST="localhost"
+    PORT=3306
+    DATABASE="Company"
+    ```
 
-```ballerina
-configurable string USER = ?;
-configurable string PASSWORD = ?;
-configurable string HOST = ?;
-configurable int PORT = ?;
-configurable string DATABASE = ?;
-```
+2. Define configurable variables to redefine the above variables and access them from within your Ballerina program.
 
->**Note:** For more information on defining configurable variables in Ballerina, see [Provide values to configurable variables](/learn/provide-values-to-configurable-variables/).
+    >**Info:** For more information on defining configurable variables in Ballerina, see [Provide values to configurable variables](/learn/provide-values-to-configurable-variables/).
 
-### Connect to the database
+    ![Create configurable variable](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-configurable-variable.gif)
 
-You can connect to the MySQL database by creating a client.
+    The generated configurable variables will be as follows.
 
-#### Import the required packages
+    ```ballerina
+    configurable string USER = "root";
+    configurable string PASSWORD = "rootPassword";
+    configurable string HOST = "localhost";
+    configurable int PORT = 3306;
+    configurable string DATABASE = "Company";
+    ```
 
-To import the [`mysql`](https://lib.ballerina.io/ballerinax/mysql/latest) 
-and [`sql`](https://lib.ballerina.io/ballerina/sql/latest) packages for creating the client, add the code below in the `main.bal` file.
+### Connect to the database
 
-```ballerina
-import ballerinax/mysql;
-import ballerina/sql;
-```
+You can connect to the MySQL database by creating a client.
 
 #### Create the MySQL client
 
-To use the `mysql:Client` to the database, add the code below in the `main.bal` file.
+Create a `mysql:Client` to connect to the database, as shown below.
+
+![Create the client](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/create-the-client.gif)
 
 ```ballerina
-final mysql:Client dbClient = check new(
-    host=HOST, user=USER, password=PASSWORD, port=PORT, database="Company"
-);
+final mysql:Client dbClient = check new (host = HOST, user = USER, password = PASSWORD, database = "Company", port = PORT);
 ```
-
 #### Run the MySQL client
 
-Execute the command below to run the client.
+Use the [**Run**](/learn/vs-code-extension/run-a-program/) option of the VS Code extension to build and run the client, as shown below.
+
+![Run the client](/learn/images/featured-scenarios/build-a-data-service-in-ballerina/run-the-client.gif)
+
+If the program runs without throwing an error with the output below, that indicates that the connection has been established successfully. This client can be defined globally and used across all the parts of the program.
 
 ```
-$ bal run
-```
+Compiling source
+        foo/data_service:0.1.0
```

Comment:
Need to use an appropriate org name. For tutorials we used integration_tutorials.

Reply:
Changed to `featured_scenarios`.